File: Array/sorting/bubblesort.py
def bubblesort(a,n):
    count = 1
    while count < n:
        for i in range(0,n-count):
            if a[i] > a[i+1]:
                a[i], a[i+1] = a[i+1], a[i]
            else:
                pass
        count +=1

    return a

if __name__ == '__main__':
    a = [6,7,1,9,2]
    n = len(a)

    temp = bubblesort(a,n)

    print(temp)

File: Array/sorting/bubblesort.cpp
// bubble sort
#include <vector>
#include  <stdio.h> 
#include <iostream>
using namespace std;

void bubbleSort(vector<int>& arr, int n)
{   
    int count = 1;
    while (count < n){
        for (int i =  0; i < (n-count); i++){
            if (arr[i] > arr[i+1]){
                swap(arr[i], arr[i+1]);
            }
            
        }
        count +=1;
    }
}

int main(){

    vector<int> arr = {6,9,4,1,5,4};
    int n = 6;
    bubbleSort(arr, n);

    for (int i=0; i < n; i++){
        cout<<arr[i]<<endl;
    }
}